pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -128,31 +128,42 @@ func (a *componentAccessorImpl) Annotations() map[string]string {
 }
 
 func (a *componentAccessorImpl) Tolerations() []corev1.Toleration {
-	tols := a.ClusterSpec.Tolerations
-	tols = append(tols, a.ComponentSpec.Tolerations...)
+	tols := a.ComponentSpec.Tolerations
```

Comment:
So the component tolerations just override the cluster level tolerations? Better add a document for this method. For it's different from other fields like node selectors.

Reply:
Yes, the field comment reveals this:

```
	// Tolerations of the component. Override the cluster-level tolerations if non-empty
	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
```

After we migrate from helm to API object, I plan to generate the api document based on the openapi schema, which has those comments information.

Reply:
I mean users calling this function can know what this fuction does.